src/__tests__/components/Article.test.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+import { Article, mapStateToProps, mapDispatchToProps } from '../../components/Article/Article';
+import { articleData } from '../../__mocks__/dummyData';
+import initialState from '../../redux/initialState.json';
+
+let wrapper;
+const props = {
+  loading: true,
+  article: articleData,
+  currentUser: {
+    username: 'username',
+  },
+  getArticle: jest.fn().mockImplementation(() => Promise.resolve({ status: 200 })),
+  rateArticle: jest.fn().mockImplementation(() => Promise.resolve({ status: 200 })),
+  onShare: jest.fn().mockImplementation(() => Promise.resolve({ status: 200 })),
+  match: {
+    params: {
+      articleSlug: 'article-slug',
+    },
+  },
+  history: { push: jest.fn() },
+};
+
+describe('<Article />', () => {
+  beforeEach(() => {
+    wrapper = mount(<Article {...props} />);
+  });
```

Comment:
@NkFab Can you add an empty line after this line for better readability?

Reply:
Done